gateway/configs/config.yaml

```diff
@@ -263,4 +263,56 @@ policy_engine:
     # Log format: json, text
     format: json
 
-policy_configurations: {}
\ No newline at end of file
+policy_configurations: {}
+
+analytics:
+  enabled: true
+  # Publishers specific configuration
+  publishers:
+    - type: moesif
+      enabled: true
+      settings: 
+        # Moesif application ID
+        application_id: <MOESIF_APPLICATION_ID>
+        # Publish interval in seconds
+        publish_interval: 5
+        # Max number of events held in memory before dropping
+        event_queue_size: 10000
+        # Max number of events in a batch
+        batch_size: 50
+        # Time interval in seconds to wake up the timer
+        timer_wakeup_seconds: 3
+
+  # gRPC Access Log Service configuration for router(Envoy)
+  grpc_access_logs:
+    # ALS Server hostname
+    host: policy-engine
```

Comment:
Envoy anyway needs the policy engine host for ext proc. We can reuse the same config. We can do this in a separate PR.